make sure that provided Cereal dependency is used + disable quiet mode when looking for dependencies via find_package
author: Kenneth Hoste (HPC-UGent)
--- openMVG/src/CMakeLists.txt.orig	2022-10-31 16:49:36.315179732 +0100
+++ openMVG/src/CMakeLists.txt	2022-10-31 16:50:11.897544247 +0100
@@ -170,9 +170,9 @@
 # ==============================================================================
 # IMAGE IO detection
 # ==============================================================================
-find_package(JPEG QUIET)
-find_package(PNG QUIET)
-find_package(TIFF QUIET)
+find_package(JPEG)
+find_package(PNG)
+find_package(TIFF)
 
 # Folders
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
@@ -235,8 +235,8 @@
 # - external by default
 # - internal if cereal not found
 # ==============================================================================
-find_package(cereal QUIET CONFIG)
-if (NOT TARGET cereal)
+find_package(cereal)
+if (NOT cereal_FOUND)
   add_library(cereal INTERFACE)
   target_include_directories(cereal
     INTERFACE
@@ -255,7 +255,7 @@
 # - external if EIGEN_INCLUDE_DIR_HINTS is defined
 # - internal if Eigen not found
 # ==============================================================================
-find_package(Eigen3 QUIET)
+find_package(Eigen3)
 if (NOT Eigen3_FOUND)
   set(EIGEN_INCLUDE_DIR_HINTS ${CMAKE_CURRENT_SOURCE_DIR}/third_party/eigen)
   set(OpenMVG_USE_INTERNAL_EIGEN ON)
@@ -270,7 +270,7 @@
 # - external by default if CERES_DIR_HINTS or find_package found a valid Ceres
 # - internal if ceres not found (ceres-solver+cxsparse+miniglog)
 # ==============================================================================
-find_package(Ceres QUIET HINTS ${CERES_DIR_HINTS})
+find_package(Ceres HINTS ${CERES_DIR_HINTS})
 if (NOT Ceres_FOUND)
   set(OpenMVG_USE_INTERNAL_CERES ON)
   set(CERES_INCLUDE_DIRS
@@ -292,7 +292,7 @@
   set(FLANN_INCLUDE_DIR_HINTS ${CMAKE_CURRENT_SOURCE_DIR}/third_party/flann/src/cpp)
   set(OpenMVG_USE_INTERNAL_FLANN ON)
 endif()
-find_package(Flann QUIET)
+find_package(Flann)
 if (NOT FLANN_FOUND OR OpenMVG_USE_INTERNAL_FLANN)
   set(FLANN_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/third_party/flann/src/cpp)
 endif()
@@ -310,7 +310,7 @@
   set(COINUTILS_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/osi_clp/CoinUtils/src/)
   set(COINUTILS_LIBRARY lib_CoinUtils)
 else()
-  find_package(CoinUtils QUIET)
+  find_package(CoinUtils)
 endif()
 
 ## ==============================================================================
@@ -328,7 +328,7 @@
      ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/osi_clp/Clp/src/OsiClp/)
   set(CLP_LIBRARIES lib_clp lib_OsiClpSolver)
 else()
-  find_package(Clp QUIET)
+  find_package(Clp)
 endif()
 
 # ==============================================================================
@@ -344,7 +344,7 @@
   set(OSI_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/osi_clp/Osi/src/Osi/)
   set(OSI_LIBRARY lib_Osi)
 else()
-  find_package(Osi QUIET)
+  find_package(Osi)
 endif()
 
 # ==============================================================================
@@ -368,7 +368,7 @@
   set(LEMON_INCLUDE_DIR_HINTS ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lemon)
   set(OpenMVG_USE_INTERNAL_LEMON ON)
 endif()
-find_package(Lemon QUIET)
+find_package(Lemon)
 if (NOT LEMON_FOUND OR OpenMVG_USE_INTERNAL_LEMON)
   set(LEMON_INCLUDE_DIRS
     ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lemon
@@ -381,7 +381,7 @@
 # - only external and enabled only if OpenMVG_USE_OPENCV is set to ON
 # ==============================================================================
 if (OpenMVG_USE_OPENCV)
-  find_package( OpenCV QUIET )
+  find_package( OpenCV )
   if (NOT OpenCV_FOUND OR OpenCV_VERSION VERSION_LESS "3.0.0")
     message(STATUS "OpenCV was not found (note that OpenCV version >= 3.0.0 is required). -> Disabling OpenCV support.")
     UPDATE_CACHE_VARIABLE(OpenMVG_USE_OPENCV OFF)
